client/me/purchases/manage-purchase/index.jsx

```diff
@@ -314,12 +315,18 @@ class ManagePurchase extends Component {
 			);
 		}
 
+		const registrationAgreementUrl = getDomainRegistrationAgreementUrl( purchase );
+		const domainRegistrationAgreementLinkText = translate( 'Domain Registration Agreement' );
+
 		return (
 			<div className="manage-purchase__content">
 				<span className="manage-purchase__description">{ description }</span>
 				<span className="manage-purchase__settings-link">
 					<ProductLink purchase={ purchase } selectedSite={ site } />
 				</span>
+				{ registrationAgreementUrl && (
+					<a href={ registrationAgreementUrl }>{ domainRegistrationAgreementLinkText }</a>
```

Comment:
Could you add the usual `target="_blank" rel="noopener noreferrer"` stuff?

Reply:
Yep.  Totally forgot about that.  Thanks!